modules/ROOT/pages/getting-started.adoc

```diff
@@ -1,142 +1,67 @@
 
-.Example FCC adding an SSH key to the core user
-[source,yaml]
-----
-variant: fcos
-version: 1.0.0
-passwd:
-  users:
-    - name: core
-      ssh_authorized_keys:
-        - ssh-rsa AAAA
-----
+TIP: The above example assumes that you have created the fcos_user on your local machine. If you did not, replace fcos_user with the user that will boot FCOS on the local host.
 
-Then use FCCT to convert the FCC to an Ignition config:
+To boot FCOS with the Ignition file, you must specify the Ignition file to the hypervisor. In the case of QEMU, do so with the `-fw_cfg` parameter. This sets the `opt/com.coreos/config` key in the QEMU firmware configuration device.
 
-.Using FCCT to convert from an FCC to an Ignition config
-[source,bash]
-----
-./bin/amd64/fcct --strict --pretty --input your_config.fcc --output your_config.ign
-----
+NOTE: You must use absolute file paths in the command below. QEMU cannot resolve relative file paths for these options.
 
-Then launch or install Fedora CoreOS with that config as described above and check that you can SSH in:
+.Example launching Fedora CoreOS with QEMU
 
-.SSHing in as the `core` user
 [source,bash]
 ----
-ssh core@<ip>
-----
-
-Refer to the FCCT https://github.com/coreos/fcct/blob/master/docs/getting-started.md[getting started guide], https://github.com/coreos/fcct/blob/master/docs/examples.md[examples] and https://github.com/coreos/fcct/blob/master/docs/configuration-v1_0.md[configuration specification] for more information on using FCCT, more examples, and a complete list of configuration FCCT supports.
-
-=== Running containers
-
-Fedora CoreOS ships with both docker and podman installed. Use systemd units to start and stop containers.
-
-.Example for running busybox using systemd and podman
-[source,yaml]
-----
-variant: fcos
-version: 1.0.0
-systemd:
-  units:
-    - name: hello.service
-      enabled: true
-      contents: |
-        [Unit]
-        Description=MyApp
-        After=network-online.target
-        Wants=network-online.target
-
-        [Service]
-        TimeoutStartSec=0
-        ExecStartPre=-/bin/podman kill busybox1
-        ExecStartPre=-/bin/podman rm busybox1
-        ExecStartPre=/bin/podman pull busybox
-        ExecStart=/bin/podman run --name busybox1 busybox /bin/sh -c "trap 'exit 0' INT TERM; while true; do echo Hello World; sleep 1; done"
-
-        [Install]
-        WantedBy=multi-user.target
+qemu-system-x86_64 -machine accel=kvm -m 2048 -cpu host -nographic \
+	-drive if=virtio,file=path/to/fedora-coreos-qemu.qcow2 \
+	-device virtio-rng-pci \
+	-fw_cfg name=opt/com.coreos/config,file=path/to/example.ign
 ----
+Once FCOS boots, it will display the IP address of the VM.
 
-Note that when using docker instead of podman, units starting docker containers will need `Requires=docker.service` and `After=docker.service` to ensure the docker daemon is running before trying to start containers.
-
-=== Running etcd
+You can then SSH into the FCOS VM as `fcos_user`:
```

Comment:
Can we skip this part here?

Rationale is:
 * the default user is `core` (this can confuse new users)
 * to SSH (on qemu) you first have to setup an SSH key
 * SSHing to machines is not the primary goal here

Reply:
If you drop this, you basically give people no way to play with the environment.

Reply:
Fair point, this is a getting-started page after all.

Then I think we can spell out the JSON content of `example.ign` here, setting up an SSH key for the `core` user. At that point, `ssh core@...` should work for playing around. Makes sense?